net/interfaces/interfaces.go

```diff
@@ -136,7 +136,7 @@ func (i Interface) IsLoopback() bool { return isLoopback(i.Interface) }
 func (i Interface) IsUp() bool       { return isUp(i.Interface) }
 
 // ForeachInterfaceAddress calls fn for each interface's address on the machine.
```

Comment:
clarify that these are not IPPrefixes in Masked (https://pkg.go.dev/inet.af/netaddr#IPPrefix.Masked) form and that the IP is the actual specific IP within that prefix?